### PR TITLE
Add a script to populate data libraries

### PR DESCRIPTION
**Pull request by bgruening**
I would like to ship this inside our Docker container so we can use this to get training data into training instances during build time.

### CODE REVIEW

File: ephemeris/library_data_example.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+libraries:
+    - name: library_name_test_1
+      files:
+        - https://zenodo.org/record/60520/files/dbSNP_138.hg19.vcf
+    - name: library_name_test_2
+      files:
```
Comment (mvdbeek) [inline ephemeris/library_data_example.yaml]: @bgruening can you call this urls? Just in case we extend this to (local) files at some point.

**Comment by mvdbeek** [comment]
Nice 👍

File: ephemeris/library_data_example.yaml (hunk 1 above)
Comment (bgruening) [inline ephemeris/library_data_example.yaml]: Interesting idea, but in this case we could automatically figure out the local file and do not need to distinguish between local and url, isn't it?

```
if os.path.exists():
    cp ...
else:
   wget ...
```
A folder would be an other idea, in this case I actually like the `files` and `folder` naming. What do you think?

Comment (mvdbeek) [inline ephemeris/library_data_example.yaml]: Yeah, that's true. OK then!

**Comment by bgruening** [comment]
Mh, tests are failing, are they supposed to work?

**Comment by mvdbeek** [comment]
Yes, these are all lint problems:
https://travis-ci.org/galaxyproject/ephemeris/jobs/162566087#L192

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**